airflow-ctl/docs/installation/prerequisites.rst

```diff
@@ -18,16 +18,16 @@
 Prerequisites
 -------------
 
-Airflow CTL is tested with:
+airflowctl is tested with:
```

Comment:
Those prerequisites are also rather high for airflowctl :) 4GB for a CLI command :) ?

Reply:
<img width="1829" height="267" alt="Screenshot from 2025-08-23 01-25-53" src="https://github.com/user-attachments/assets/570ccc27-e209-49f7-957a-d82ba9d59ddb" />

I updated 200 MB to be safe, doubled the number, but still should be good :)